app/sitemap.ts

```diff
@@ -19,8 +19,8 @@ export default async function sitemap(): Promise<MetadataRoute.Sitemap> {
     cv: Date.now(),
   });
 
-  // Exclude any stories with noindex set to true on Storyblok
-  const indexStories = response.filter((story) => !story.content?.noindex);
+  // Exclude any stories with noindex set to true and those inside the Global Components or Test folders in Storyblok
+  const indexStories = response.filter((story) => (!story.content?.noindex) && !story.full_slug?.startsWith('global-components/') && !story.full_slug?.startsWith('test/'));
```

Comment:
@sherakama I'm doing some final clean up of this PR. To have the masthead show up, I"ll need to publish the Default Masthead on Storyblok that's inside the /global-components/mastheads folder. I'm removing it from the sitemap here, and also disallow it in robots.ts. Is that enough, or do I need anything else? Ideally people can't go to https://momentum.stanford.edu/global-components/mastheads/default-masthead, what would be your recommended way of doing this? 🤔

Reply:
This is not the place I would recommend doing this. You can adjust the filter query on the request in lines `17-19` of this file

https://www.storyblok.com/docs/api/content-delivery/v2/filter-queries/operation-not-in

something like

```
const response = await storyblokClient.getAll('cdn/stories', {
    version: 'published',
    cv: Date.now(),
		filter_query: {
      component: {
        not_in: ['Masthead', 'Story List Nav']
      }
    }
  });
```

or 

https://www.storyblok.com/docs/api/content-delivery/v2/filter-queries/operation-not-like

```
const response = await storyblokClient.getAll('cdn/stories', {
    version: 'published',
    cv: Date.now(),
		filter_query: {
      slug: {
        not_like: "test*"
      }
    }
  });
```

Reply:
However, 

We are doing something similar in other places

https://github.com/SU-SWS/ood_online_giving/pull/211/files#diff-68c980f6cc36311701d8a04a11d2bd6363c91b7199c333615e4d33b26f2e20e1

Reply:
Oh dang - thanks for sharing the filter query! How did I miss that? 😁 I swear I was reading the doc. That was what I was looking for. Looks like a good approach to me.

Reply:
@sherakama Interesting - it not_in doesn't seem to like multiple values at all, and not_like doesn't appear to be working for me 🤔 Are you ok with me pushing what I have to prod, and then we'll work on this together after I come back?

Reply:
not_in works when I have a single component I want to exclude, eg, `not_in: 'sbStoryMvp"` works as expected, but it doesn't seem to be able to handle an array, or strings separated by commas (it will ignore the item after the comma for me)